activestorage/lib/active_storage/service/s3_service.rb

```diff
@@ -104,6 +94,17 @@ def headers_for_direct_upload(key, content_type:, checksum:, filename: nil, disp
     end
 
     private
+      def private_url(key, expires_in:, filename:, disposition:, content_type:)
```

Comment:
Why do `public_url` and `private_url` overrides sometimes need a keyword splat at the end (`**`) and sometimes not?

Reply:
Thanks, fixed so that they all have splats at the end.

Reply:
Thanks, fixed so that they all have splats at the end.